numpy/_core/src/multiarray/item_selection.c

```diff
@@ -1038,7 +1038,7 @@ PyArray_Choose(PyArrayObject *ip, PyObject *op, PyArrayObject *out,
                                                     multi->dimensions,
                                                     NULL, NULL, 0,
                                                     (PyObject *)ap);
-        out_newarr = 1;
+        copy_out = 1;
```

Comment:
```suggestion
```

Reply:
Oh yes, your naming suggestion makes total sense now 😓 I'll change that back too, thanks!